lib/patch-series.ts

```diff
@@ -624,6 +623,8 @@ export class PatchSeries {
                 throw new Error(`Could not extract cover letter Message-ID`);
             }
 
+            const timeStamp = coverMid.match(/cover\.([0-9]+)\./);
+            const midTStamp = timeStamp ? timeStamp[1] : `${Date.now()}`;
```

Comment:
Hmm. I really had hoped to avoid two variables here, and certainly I see still the possible confusion between them given that they have very, very similar names. I guess I could warm up to calling the first one `match` and the second one `timeStamp` instead.

Reply:
As noted earlier, two vars are needed due to different types.  `match` was already used so `tsMatch` was chosen.

Reply:
Ah, okay!